src/components/movies/MoviesCards.tsx

```diff
@@ -20,64 +20,75 @@ import { Link } from 'react-router-dom';
 
 //faire en sorte que se soit de manière dynamique en front et mettre jsute une 1s d'attente ou pas
 
-function RenderCards({ showClassic, movie, userMovie }: { showClassic: boolean; movie: Movie; userMovie?: UserMovie[] }) {
-  
+function RenderCards({
+  showClassic,
+  movie,
+  //userMovie,
+}: {
+  showClassic: boolean;
+  movie: Movie;
+  userMovie?: UserMovie[];
```

Comment:
userMovie n'a pas l'air utilisé ici, tu peux le retirer
```suggestion
}: {
  showClassic: boolean;
  movie: Movie;
```